README.md

```diff
@@ -94,6 +94,20 @@ This repo contains images specific to ROS 2, and are particularly for developmen
   - **Notice:**
     - includes third party license agreements for non free software
     - including the [`Open Community Source`](https://www.rti.com/products/pricing/compare) license from RTI
+- `devel`
+  - _Description:_
+    - includes pre-installed dependencies for building from source
+    - closely mimics the source install setup instructions 
+  - _Purpose:_
+    - intended for development and bootstrapping source builds
+    - small image size and kept freash by rebuilding every day
+- `source`
```

Comment:
Do we want do dissociate the tags that are available on DockerHub vs the docker files that don't have any image published?
e.g. all the `nightlies` and `devel` are built and tagged regularly but source is more of a reference dockerfile that never gets build ATM. It's mostly used as a starting point for people that want to then integrate static analysis, custom build options etc

Reply:
Added notice about hosting of image.